app/assets/javascripts/admin/listing_shapes.js

```diff
@@ -116,15 +141,37 @@ window.ST.initializeListingShapeForm = function(formId) {
     this.parentElement.remove();
   };
 
-  var toggle = function(el, state) {
+  var toggleCheckbox = function(el, state) {
```

Comment:
I don't understand this function. Is it only there to enable/disable the checkbox for user to check/uncheck, and unchecks when disabled to avoid submitting the value?

The name leads me to think it would toggle the value, not the disabled/enabled state.